lib/high_voltage/page_finder.rb

```diff
@@ -31,7 +31,13 @@ def clean_path
     end
 
     def clean_id
-      @page_id.tr("^#{VALID_CHARACTERS}", '')
+      @page_id.tr("^#{VALID_CHARACTERS}", "").tap do |id|
+        unless id.length > 0
+          raise InvalidPageId.new "Invalid page id: #{@page_id}"
+        end
+      end
     end
   end
+
+  class InvalidPageId < Exception; end
```

Comment:
Can we rename this to `InvalidPageIdError` and inherit from `StandardError` instead? Typically custom error classes should inherit from `StandardError` instead of `Exception`. See http://www.codeotaku.com/journal/2009-08/ruby-standard-error/index for more details.